tests/test_api.py

```diff
@@ -15,6 +15,11 @@
 from .common import Point, Point3D, i64, f64, to_abstract_test, ai64_of, \
     ai32_of, af64_of
 
+from myia.abstract.infer import ArrayWrapper
+from myia.compile import load_backend
+
+from dataclasses import dataclass
```

Comment:
Put stdlib imports at the top, and imports from myia above the local import from `.common`.